testing.rst

```diff
@@ -567,6 +567,56 @@ command.
     If the information you need to check is available from the profiler, use
     it instead.
 
+Simulating Login
```

Comment:
I'd reword the title as `User Login` (or `Loging in Users` ?) ("simulating" seems a bit excessive, given that most of the times, testing is about "simulating" things)